docs/content/en/docs/release-notes/_index.md

```diff
@@ -0,0 +1,95 @@
+
+---
+title: "K8ssandra release notes"
+linkTitle: "Release notes"
+weight: 1
+description: Release notes for the open-source K8ssandra community project.
+---
+
+K8ssandra provides a production-ready platform for running Apache Cassandra&reg; on Kubernetes. This includes automation for operational tasks such as repairs, backup and restores, and monitoring. 
+
+Also deployed is Stargate, an open source data gateway that lets you interact programmatically with your Kubernetes-hosted Cassandra resources via a well-defined API. 
+
+The **K8ssandra 1.1.0** release implements a number of key changes, enhancements, and bug fixes, as noted below. For the complete list, see [CHANGELOG-110.md](https://github.com/k8ssandra/k8ssandra/blob/main/CHANGELOG.md).  
+
+**Release date:** 09-April-2021.
+
+## Prerequisites
+
+* A Kubernetes v1.15 or later environment - local or via a supported cloud provider
+* [Helm](https://helm.sh/) v3
+
+## Supported Kubernetes environments
+
+* Open-source [kubernetes.io](https://kubernetes.io)
+* [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine) (GKE)
+* [Microsoft Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service/) (AKS)
+* [Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks/) (EKS)
+* [MiniKube](https://minikube.sigs.k8s.io/docs/)
+* [Kind](https://kind.sigs.k8s.io/)
+* [K3D](https://k3d.io/)
+
+## K8ssandra deployed components
+
+The K8ssandra helm chart deploys the following components. Some are optional, and depending on the configuration, may not be deployed:
+
+* Apache Cassandra  the deployed version depends on the configured `serverImage` setting:
+  * 3.11.7
+  * 3.11.8
+  * 3.11.9
+  * 3.11.10 (default)
+* DataStax Kubernetes Operator for Apache Cassandra (cass-operator) 1.6.0
+* Management API for Apache Cassandra 0.1.19
```

Comment:
The management-api version we are deploying is 0.1.24. The management-api image includes MCAC. I believe the version of MCAC is 0.2.0. @emerkle826 can you please confirm?

Reply:
Checking with Erik @emerkle826..

Reply:
yes, MCAC (Metrics Collector for Apache Cassandra) version 0.2.0 is what comes with MAAC (Management API for Apache Cassandra) version 0.1.24

Reply:
Thanks - updated with latest commit.